pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -170,36 +170,36 @@ func initHandler() {
 	}
 }
 
-func getInterfaceCacheFile(name string) string {
-	return fmt.Sprintf(interfaceCacheFile, name)
-}
-
-func setCachedInterface(name string, ifconf *api.Interface) error {
-	buf, err := json.MarshalIndent(&ifconf, "", "  ")
+func writeToCachedFile(inter interface{}, fileName, name string) error {
+	buf, err := json.MarshalIndent(&inter, "", "  ")
 	if err != nil {
 		return fmt.Errorf("error marshaling interface cache: %v", err)
 	}
-	err = ioutil.WriteFile(getInterfaceCacheFile(name), buf, 0644)
+	err = ioutil.WriteFile(getInterfaceCacheFile(fileName, name), buf, 0644)
 	if err != nil {
 		return fmt.Errorf("error writing interface cache %v", err)
 	}
 	return nil
 }
 
-func getCachedInterface(name string) (*api.Interface, error) {
-	buf, err := ioutil.ReadFile(getInterfaceCacheFile(name))
+func readFromCachedFile(name, fileName string, inter interface{}) (bool, error) {
+	buf, err := ioutil.ReadFile(getInterfaceCacheFile(fileName, name))
 	if err != nil {
 		if os.IsNotExist(err) {
-			return nil, nil
+			return false, nil
 		}
-		return nil, err
+		return false, err
 	}
-	ifconf := api.Interface{}
-	err = json.Unmarshal(buf, &ifconf)
+
+	err = json.Unmarshal(buf, &inter)
 	if err != nil {
-		return nil, fmt.Errorf("error unmarshaling interface: %v", err)
+		return false, fmt.Errorf("error unmarshaling command line: %v", err)
```

Comment:
Command line? I think you we use these functions in two contexts: for qemu CLI args and for interface cache; it would be incorrect to log this message for interface cache; (same issue but vice versa in write function).